app/views/bootstrap/footer.scala.html

```diff
@@ -41,7 +41,7 @@ <h5>Our Sponsors</h5>
         <div class="container">
             <div class="row">
                 <div class="col-lg-12">
-                    <p>Copyright &copy; @messages("general.organization") 2016-2018</p>
+                    <p>Copyright &copy; @messages("general.organization") 2016-2019</p>
```

Comment:
are you able to make this be the current year so it doesn't need to be changed again?

Reply:
Fixed in latest commit